x-pack/plugins/security_solution/public/kubernetes/pages/translations.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const COLUMN_SESSION_START = i18n.translate(
+  'xpack.securitySolution.kubernetes.columnSessionStart',
+  {
+    defaultMessage: 'Date connected',
+  }
+);
+
+export const COLUMN_EXECUTABLE = i18n.translate(
+  'xpack.securitySolution.kubernetes.columnExecutable',
+  {
+    defaultMessage: 'Session leader',
+  }
+);
+
+export const COLUMN_NODE = i18n.translate('xpack.securitySolution.kubernetes.columnNode', {
+  defaultMessage: 'Node',
+});
+
+export const COLUMN_ENTRY_USER = i18n.translate(
+  'xpack.securitySolution.kubernetes.columnEntryUser',
+  {
+    defaultMessage: 'Session entry users',
```

Comment:
@nick-alayil suggested to have it match the title of the session chart https://github.com/elastic/kibana/pull/134535